### ISSUE TITLE
5fold and 1fold experiment GAP

### ISSUE THREAD

**Issue by huangyanzhou163** [padeoe/cail2019#9]
Hi, I run your code under such setting:
TRAINING_DATASET = 'data/raw/CAIL2019-SCM-big/SCM_5k.json'
BERT_PRETRAINED_MODEL=民事文书BERT
config = {
        "max_length": 512,
        "epochs": 2,
        "batch_size": 12,
        "learning_rate": 2e-5,
        "fp16": False
    }
trainer.train(MODEL_DIR, 5)
The Acc in the test set 85%. 
However, if I set the TRAINING_DATASET = 'data/train/input.txt' and trainer.train(MODEL_DIR, 1), then the Acc in the test set is 66.6%. I don't know why there is such a big gap between the 5fold and 1fold. 
P.S. I both copy the original vocab.txt from the BERT_PRETRAINED_MODEL in 5fold and 1fold experiment in case of the "vocabulary indices are not consecutive".

I guess that training set in the 5th fold including many instances in the test set (create by data.py). So the test Acc can reach  85%. So I believe the 85% may not be solid.

**Comment by padeoe** [comment]
When k=1, It will not perform k-fold cross validation, the fixed training dataset and test dataset specified in `main` will be used.
```python
# TRAINING_DATASET = 'data/train/input.txt'  # for quick dev
TRAINING_DATASET = 'data/raw/CAIL2019-SCM-big/SCM_5k.json'

test_input_path = 'data/test/input.txt'
test_ground_truth_path = 'data/test/ground_truth.txt'
```
The default training data is all data, so all test data has been seen in training. I do this because it's for final submit. You can just comment out `TRAINING_DATASET = 'data/train/input.txt'` so the test data is never been seen in training when k=1, or you can use k bigger than 1,then it will perform a real k-fold.

**Comment by huangyanzhou163** [comment]
I have already tried TRAINING_DATASET = 'data/train/input.txt' and set K=1. And the result is 0.66. Is this result a normal result? If it does, what make this happen? The limited size of the training data or the problem is very hard?
